x-pack/elastic-agent/pkg/core/monitoring/beats/beats_monitor.go

```diff
@@ -113,7 +113,6 @@ func (b *Monitor) EnrichArgs(process, pipelineID string, args []string, isSideca
 		logFile = fmt.Sprintf("%s-json.log", logFile)
 		appendix = append(appendix,
 			"-E", "logging.json=true",
-			"-E", "logging.ecs=true",
```

Comment:
I am confused? If this is removed then what is added the ecs.version to the events?

Reply:
confused as well i'm still looking, but i think this is not ideal place to fix, need to play with it a bit more

Reply:
i think that this is due to monitoring beat with a beat so we monitor data which already contains ecs flags and injects additional one there using ^ settings. 
One ECS is probably comming from processing pipeline (look for `WithECS`) and additional one we were adding when logging.

This is why i see 2 versions 1.5.0 and 1.6.0. `1.6.0` is coming from publishing pipeline and version is set in libbeat.
`1.5.0` is coming from logp as it is hardcoded there. 

And as i dont want to alter publishing pipeline and libbeat code, this seems like a good place for a fix after all.

Reply:
I think it's a problem with logp, that version should be centralized. WDYT @urso

Reply:
Ah, that explains why. Agent is already adding the ecs fields in the log, then filebeat is adding it again.

Reply:
Does enabling the ecs flags do more than just adding the _version_?

Reply:
i compared event with ecs enabled and without ecs and they differed in version, `host` and `agent` stayed there